README.md

```diff
@@ -1,2 +1,38 @@
-# tax-measures-website
-VTA Tax Measures Single Page Application - Developed by Blinktag
+# VTA Tax Measures Single Page Application
+
+## Setup
+
+Requires node.js and yarn to be installed.
+
+Install node.js
+
+https://nodejs.org/
+
+Install Yarn
+
+https://yarnpkg.com/
+
+Install dependencies
+
+    yarn
+
+## Running locally
+
+Start the development server:
+
+    yarn dev
+
+Open http://localhost:3000 in your browser
+    
+## Building locally
+
```

Comment:
Might also want to mention `yarn install`

Reply:
I updated the readme to mention that setup needs to happen before building locally.